### PR TITLE
Policy setting from config file

### PR DESCRIPTION
**Pull request by chr1sgrant**

### CODE REVIEW

File: antha/anthalib/wtype/LiquidType.go [hunk 1]
```diff
@@ -18,213 +18,67 @@ func PolicyNameFromString(s string) PolicyName {
 	return PolicyName(s)
 }
 
-type LiquidType int
+// LiquidType represents the type of a Liquid
```
Comment (m-i-sadowski) [inline antha/anthalib/wtype/LiquidType.go]: I love this comment

File: microArch/driver/liquidhandling/robotinstruction.go [hunk 2]
```diff
@@ -25,9 +25,12 @@ package liquidhandling
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"sort"
+	"strings"
 
+	"github.com/antha-lang/antha/antha/AnthaStandardLibrary/Packages/text"
```
Comment (m-i-sadowski) [inline microArch/driver/liquidhandling/robotinstruction.go]: Sorry, got to insist you remove this please

**Comment by chr1sgrant** [comment]
This is up to date now

**Comment by chr1sgrant** [comment]
brings in liquidtype package dependency so therefore desirable

**Pull request merged**